src/project/internal/notationproject.cpp

```diff
@@ -658,6 +665,31 @@ void NotationProject::setMetaInfo(const ProjectMeta& meta)
     score->setMetaTags(tags);
 }
 
+void NotationProject::setMetaInfoUndoable(const ProjectMeta& meta)
```

Comment:
There is no need for this method. You can use NotationProject::setMetaInfo. You just need to add a bool argument to it (and improve the method accordingly):

 NotationProject::setMetaInfo(const ProjectMeta& meta, bool undoable = false)